conf/celeborn-env.sh.template

```diff
@@ -32,4 +32,4 @@
 # CELEBORN_MASTER_JAVA_OPTS="-XX:-PrintGC -XX:+PrintGCDetails -XX:+PrintGCTimeStamps -XX:+PrintGCDateStamps -Xloggc:gc-master.out -Dio.netty.leakDetectionLevel=advanced"
 # CELEBORN_PID_DIR="$CELEBORN_HOME/pids"
 # CELEBORN_LOG_DIR="$CELEBORN_HOME/logs"
-# DISABLE_JEMALLOC="false"
+# CELEBORN_PREFER_JEMALLOC="true"
```

Comment:
can we list `CELEBORN_JEMALLOC_PATH` in the example too? additionally, they should be listed at L26 too.